rmf_fleet_adapter/src/full_control/main.cpp

```diff
@@ -467,11 +467,26 @@ struct Connections : public std::enable_shared_from_this<Connections>
           path_request_pub, mode_request_pub);
 
     const auto& l = state.location;
+    const auto& starts = rmf_traffic::agv::compute_plan_starts(
+        *graph, state.location.level_name, {l.x, l.y, l.yaw},
+        rmf_traffic_ros2::convert(adapter->node()->now()));
+    if (starts.empty())
+    {
+      RCLCPP_ERROR(
```

Comment:
I believe this will end up getting printed every time a new state update comes in, until the problem is resolved. It might be good to have something keep track of which robots this message has been printed out for, and only do it once.

Reply:
Actually, it will only be printed once as the `Connections::add_robot()` function is only called the first time the `RobotState` message for that robot is received. I've tested this by having the `slotcar` continuously publish `RobotState` msgs with invalid `level_name`. 
https://github.com/open-rmf/rmf_ros2/blob/87de246ade6c371b6d594080a34b971721c55085/rmf_fleet_adapter/src/full_control/main.cpp#L746-L752

Reply:
Ah, I see.

In that case, it means if the initial state for the robot was invalid, then the robot can never be added until the fleet adapter is restarted, which also does not seem great to me. But then again, this is a legacy adapter for an API that we don't really want people to be using anyway, so I'm okay with leaving it less robust than it could be.